### PR TITLE
Flag profiles ready to expire eligible for an early renewal.

### PR DESCRIPTION
**Pull request by akatsoulas**
* minor changes to help readabiity.

### CODE REVIEW

**Comment by johngian** [comment]
Tested it locally:

* It doesn't allow you to renew membership for members
* There is no way for curators to filter only pending renewals

**Comment by akatsoulas** [comment]
@johngian pr updated. The membership of existing members can be renewed. As for the second comment, let's tackle it in another PR since it seems to me a bit out of scope.

File: mozillians/groups/models.py [hunk 1]
```diff
@@ -305,6 +309,7 @@ def add_member(self, userprofile, status=GroupMembership.MEMBER):
                         (GroupMembership.PENDING_TERMS, GroupMembership.MEMBER)]
             if (old_status, status) in statuses:
                 # Status changed
+                # let's remove the needs renewal flag
```
Comment (johngian) [inline mozillians/groups/models.py]: Please move this comment to line 295

**Comment by johngian** [comment]
@akatsoulas I pushed a patch here to allow filtering renewals please review and apply:

https://github.com/johngian/mozillians/commit/b3aedc03dbda57abab95365feda7da7cc612b54f

**Pull request merged**